guides/common/modules/proc_enabling-connections-to-capsule.adoc

```diff
@@ -7,8 +7,18 @@ On the base operating system on which you want to install {SmartProxy}, you must
 include::snip_firewalld.adoc[]
 
 .Procedure
-
-. On the base operating system on which you want to install {SmartProxy}, enter the following command to open the ports for {ProjectServer} and clients communication to {SmartProxyServer}:
+ifdef::katello,satellite,orcharhino[]
+. Open the ports for clients on {ProjectServer}:
++
+[options="nowrap"]
+----
+# firewall-cmd \
+--add-port="5647/tcp" \
```

Comment:
Like mentioned elsewhere, qpid is gone and this should also be gone. Not strictly required for this since it was already there, but something that must be fixed nonetheless.

Reply:
I can raise another PR for this :+1: